.travis/shellchecker

```diff
@@ -79,12 +79,13 @@ default () {
     main . \
         --exclude etc/rose-bash-completion \
         --exclude t \
-        -- -e SC1090
+        -- -e SC1090 -e SC2119
 
     # run a lenient check on all test scripts
     main t -- -S error -e SC1090
 }
 
+shellcheck -V
```

Comment:
I put this in for debugging, but outputting the version of shellcheck used seems like a good idea. Thoughts?

Reply:
Whatever.

If desired you could put it in the Travis "after failure" section (ps: Matt will ask you to send this output to stderr).